src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import {
+	Slider,
+	TextInput,
+	NumberInput,
+	Checkbox
+} from 'carbon-components-react';
+import { css, cx } from 'emotion';
+import { AComponent, ComponentInfo } from './a-component';
+import image from './../assets/component-icons/slider.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../tools';
+
+const preventCheckEventStyle = css`
+	pointer-events: none;
+`;
+
+export const ASliderSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<Checkbox
+			labelText='Disabled'
+			id='disable-label'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					disabled: checked
+				});
+			}} />
```

Comment:
Just an observation, the elements below (other than checkbox), you omit the curly braces but there here you don't. I think we can remove since we have no other logic other than setter.